app/src/androidTest/java/org/dhis2/usescases/event/EventIntents.kt

```diff
@@ -0,0 +1,43 @@
+package org.dhis2.usescases.event
+
+import android.content.Intent
+import androidx.test.rule.ActivityTestRule
+import org.dhis2.usescases.eventsWithoutRegistration.eventCapture.EventCaptureActivity
+import org.dhis2.usescases.eventsWithoutRegistration.eventInitial.EventInitialActivity
+import org.dhis2.usescases.teiDashboard.TeiDashboardMobileActivity
+
+const val EVENT_UID = "EVENT_UID"
+const val PROGRAM_UID = "PROGRAM_UID"
+const val TEI_UID = "TEI_UID"
+const val ENROLLMENT_UID = "ENROLLMENT_UID"
+const val PROGRAM_STAGE_UID = "PROGRAM_STAGE_UID"
+
+
+const val PROGRAM_TB_UID = "ur1Edk5Oe2n"
+const val EVENT_DETAILS_UID = "y0xoVIzBpnL"
+const val TEI_EVENT_TO_DELETE_UID = "foc5zag6gbE"
+const val ENROLLMENT_EVENT_DELETE_UID =  "SolDyMgW3oc"
+const val PROGRAM_STAGE_TO_SHARE = "EPEcjy3FWmI"
+
+fun prepareEventDetailsIntentAndLaunchActivity(rule: ActivityTestRule<EventCaptureActivity>) {
+    Intent().apply {
+        putExtra(PROGRAM_UID, PROGRAM_TB_UID)
+        putExtra(EVENT_UID, EVENT_DETAILS_UID)
+    }.also { rule.launchActivity(it) }
```

Comment:
@Sharmyn28 can you do this with XX Tracker Program instead and alpha event if it's possible? It's actually where we found the bug.

Thanks

Reply:
Of course! I'll change it and use XX Tracker Program :)